### DISCUSSION TITLE
how to restart nvim-cmp on crash without restarting nvim

### DISCUSSION THREAD

**Discussion by alexmaloteaux** [hrsh7th/nvim-cmp#1477] [Q&A]
Hi
i have nvim-cmp crashing after long usage , mainly when editing comments, 
the exact error message im getting is this one ; https://github.com/hrsh7th/nvim-cmp/issues/1097

but i cant really reproduce the bug accurately, in a nutshell, after coding several and starting a comment  , then nvim-cmp crash.

The main issue for me is that except restarting nvim , im not sure if there is any builtin to nicely restarting nvim-cmp instead as when the bug happens , nvim-cmp is disabled on all my projects (tabs).
Thanks

**Comment by arminveres** [comment]
Did you find a solution to this? cmp sometimes crashes without any warnings.

**Comment by alexmaloteaux** [comment]
no but i checked plugin by plugin which one was making nvim-cmp crash. it and up being the map_cr option of nvim-autopairs and i disabled it like this : 
```require("nvim-autopairs").setup({check_ts = true, map_cr = false})```

never had a crash since
